src/Sarif.UnitTests/Converters/AndroidStudioConverterTests.cs

```diff
@@ -345,8 +363,14 @@ public void AndroidStudioConverter_ConvertSarifResult_GeneratesLocationWithOnlyP
 
             LocationInfo locationInfo = GetLocationInfoForBuilder(builder);
 
-            locationInfo.Location.Should().Be(expectedLocation);
-            locationInfo.LogicalLocationComponents.SequenceEqual(expectedLogicalLocationComponents).Should().BeTrue();
+            LocationEqualityComparer.Instance.Equals(locationInfo.Location, expectedLocation)
+                .Should().BeTrue();
+
+            locationInfo.LogicalLocationComponents
+                .SequenceEqual(
+                    expectedLogicalLocationComponents,
+                    LogicalLocationComponentEqualityComparer.Instance)
```

Comment:
For the cases where you need the comparer (like this), consider having something a little shorter (and maybe less cluttering, and easier to find). 

Maybe:

``` C#
public class LogicalLocationComponent {
     ...
     public static IEqualityComparer<LogicalLocationComponent> ValueComparer { get; }
}
```

You could then make the concrete comparer types internal.